R/utils.R

```diff
@@ -0,0 +1,8 @@
+# Utilities
+
+read_csv_file <- function(fn) {
+  read.csv(
+    system.file("extdata", fn, package = "fortedata", mustWork = TRUE),
+    stringsAsFactors = FALSE
+  )
+}
```

Comment:
Two nit-picky comments:
- Internal functions should probably still be documented, just without the `@export` tag. We don't need a ton of detail, but it can't hurt to at least document parameters.
- I would change `fn` to `...`. That will allow us to use this with files in subdirectories.